### PR TITLE
[GROW-1731] Update docs

### PR DESCRIPTION
**Pull request by alexbwell**
Update documentation on contributing to this project.

### CODE REVIEW

**Comment by samantha-payson** [comment]
Thanks for keeping the docs up to date!

File: README.md [hunk 1]
```diff
@@ -35,9 +35,41 @@ npm start
...
 and visit http://localhost:3005/api-docs
 
+# Contributing guidelines
+All contributions of new API endpoints must follow the process below to ensure we follow the same coding style and maintaining high quality of code.
+
+## 1. Code format
+All new routes belonging to the same business domain must be grouped together under the same express router in a separate file. These routes are then imported into the main application.
+
+**Errors**
+Errors must also be logged, so they can be captured in our CloudWatch logs. Critical errors should be also logged to Sentry to alert the team in timely fashion.
```
Comment (jasonleibowitz) [inline README.md]: Is it worth explaining _how_ to log errors, i.e. using `console.error` or something else?

File: README.md [hunk 2]
```diff
@@ -35,9 +35,41 @@ npm start
...
+
+> #### 3rd Party API dependencies
+> Any such dependencies must be explicitly documented in comments directly on the route. The expected response format should also be documented.
+>
+> **Errors**
+> Any 3rd party API calls must include `catch` clause(s) in order to capture the errors without crashing the application.
+
+## 2. Tests
+All routes must be covered by unit tests and any exceptions thoroughly documented. We use `supertest` library to mock http requests.
```
Comment (jasonleibowitz) [inline README.md]: 💯

File: README.md (hunk 1 above)
Comment (alexbwell) [reply README.md]: Good call. I'll ask @kl4w and @dminnear what the best way to log stuff in order for it to get captured in CloudWatch.

**Pull request merged**